source/manual/howto-backup-and-restore-in-aws-rds.html.md

```diff
@@ -105,172 +107,45 @@ gds-cli aws govuk-<environment>-admin aws rds wait db-instance-available --db-in
-
-For example:
+### 7. Make a change to the database contents
 
-* database-name = `restored-<name_old_db>`
-* stack-name = blue
-* govuk-internal-domain = `<environment>.govuk-internal.digital`
-* restored-db-endpoint= echo ${endpoint_address} in your terminal then copy and past into file
-
-```
-{
-    "Comment": "Manual DB restore",
-    "Changes": [
-        {
-            "Action": "UPSERT",
-            "ResourceRecordSet": {
-                "Name": "<database-name>.<stack-name>.<govuk-internal-domain>",
-                "Type": "CNAME",
-                "TTL": 300,
-                "ResourceRecords": [
-                    {
-                        "Value": "<restored-db-endpoint>"
-                    }
-                ]
-            }
-        }
-    ]
-}
-```
-
-Apply these changes with the following command:
-
-```
-gds-cli aws govuk-<environment>-admin aws route53 change-resource-record-sets --hosted-zone-id ${next_hosted_zone_id} --change-batch file:///var/tmp/update_dns.json
-```
-
-The output should look something like:
-
-```
-{
-    "ChangeInfo": {
-        "Id": "/change/C1045684TR3O47QOC1T6",
-        "Status": "INSYNC",
-        "SubmittedAt": "2023-08-23T15:16:15.298000+00:00",
-        "Comment": "Manual DB restore"
-    }
-}
-```
-
-It can take a couple of minutes for the change to be applied and you might have the status `PENDING`. You can check the status by running:
-
-```
-gds-cli aws govuk-<environment>-admin aws route53 get-change --id /change/<ChangeInfo_Id>
-```
-
-The restore is now finished!
-
-Please be aware that if you changed the file path in the previous step, remember to change the code below.
-
-### 10. Check the changes in Route53
-
-To see the changes to AWS Route 53, Log into AWS Console > Route 53 > Hosted Zones > Check the Public and private link to <environment>.govuk-internal.digital and search for the restored DB.
-
-## Point the application to the new RDS instance
-
-Once you have restored a database in AWS RDS, you now need to point the corresponding app towards it.
-
-### 1. Make a change to the database contents
-
-Through the app's user interface, or via the app console or database console, make
-a change that you can use as a sense check to verify that the database switch has
-been successful.
+Through the app's user interface, or via the app console, make a change that you
+can use as a sense check to verify that the database switch has been successful.
 
 For example, you might create a draft edition of something, or modify or delete
 a record. After switching to the restored database, your changes should be undone.
 
-In this example we want to make a change to the database `local-links-manager_production`
-such as delete an old record:
+In this example we open a Rails console on local-links-manager, from which we can
+make a change such as deleting an old record:
```

Comment:
This step seems a bit vague. Not sure if I've correctly understood the intent but it sounds like it's about checking that the client app can read/write the restored database? If so, can we replace it with a more specific and straightforward check?

Could maybe delete this paragraph and the Rails console snippet and just leave the bit about about checking that the app can still write stuff by e.g. creating a draft edition.

Reply:
I think the intent was to make sure the restore was successful by making a change before the pre-restore, and then checking if the change was reverted post-restore.

It's probably easier/less confusing if I remove that whole section and suggest checking which DB ActiveRecord is connected to via the Rails console instead.